### PR TITLE
Convenience

### PR DESCRIPTION
**Pull request by edelmanjm**
Add convenience methods for working with multiple motors, extract common interface for ChickenTalon and ChickenVictor

### CODE REVIEW

File: src/main/java/org/team1540/base/ChickenSubsystem.java [hunk 1]
```diff
@@ -31,14 +35,22 @@ public boolean contains(ChickenTalon o) {
     return motors.contains(o);
   }
 
-  public boolean add(ChickenTalon motor) {
-    return motors.add(motor);
+  public boolean add(ChickenTalon o) {
```
Comment (RobinsonZ) [inline src/main/java/org/team1540/base/ChickenSubsystem.java]: Why the change in parameter name?

File: src/main/java/org/team1540/base/wrappers/ChickenTalon.java [hunk 2]
```diff
@@ -20,7 +20,7 @@
  * Wrapper around a {@link TalonSRX} adding some features (that really should already be there) as
  * well as making functions easier to call.
  */
-public class ChickenTalon extends TalonSRX {
+public class ChickenTalon extends TalonSRX implements ChickenController {
```
Comment (RobinsonZ) [inline src/main/java/org/team1540/base/wrappers/ChickenTalon.java]: Maybe run a cleanup in IntelliJ to insert `@Override`s?

File: src/main/java/org/team1540/base/ChickenSubsystem.java (hunk 1 above)
Comment (edelmanjm) [reply src/main/java/org/team1540/base/ChickenSubsystem.java]: Cuz it was confusing as was
